sys/amd64/conf/GENERIC-MMCCAM

```diff
@@ -1,33 +1,23 @@
-# MMCCAM is the kernel config for doing MMC on CAM development
-# and testing on bhyve
+#
+# GENERIC-MMCCAM
+#
+# Custom kernel for GENERIC plus MMCCAM as opposed to the prior MMC stack.
+#
 
 #NO_UNIVERSE
 
-include         MINIMAL
+include		GENERIC
 
 ident		GENERIC-MMCCAM
 
-# Access GPT-formatted and labeled root volume
-options 	GEOM_LABEL
-
-# UART -- for bhyve console
-device          uart
-
-# VirtIO support, needed for bhyve
-device		virtio			# Generic VirtIO bus (required)
-device		virtio_pci		# VirtIO PCI device
-device		vtnet			# VirtIO Ethernet device
-device		virtio_blk		# VirtIO Block device
-device		virtio_scsi		# VirtIO SCSI device
-device		virtio_balloon		# VirtIO Memory Balloon device
-
-# CAM-specific stuff
-device		pass
-device		scbus
-device		da
-
 options 	MMCCAM
 
 # Add CAMDEBUG stuff
 options 	CAMDEBUG
 options 	CAM_DEBUG_FLAGS=(CAM_DEBUG_INFO|CAM_DEBUG_PROBE|CAM_DEBUG_PERIPH)
```

Comment:
I'd drop the last of these lines. CAMDEBUG is fine.. it make it easy to turn on debugging. CAM_DEBUG_FLAGS make things too chatty by default. Yes, the one that's there should likely have that done to it too.

Reply:
removed, in its own commit, so you can decide whether that's a good idea or not as you review this PR